File: Java/Driver.java
public class Driver{
    public static void main(String[] args){
        Prime p = new Prime();
        Reverse r = new Reverse();

        for(int num = 1; num <= 100; num++){
            if(p.isPrime(num)){
                int revNum = r.getReverse(num);
                if(p.isPrime(revNum)) System.out.println(num);
            }
        }
    }
}

File: C/Driver.c
#include <stdio.h>
#include "Prime.h"
#include "Reverse.h"
int main(){
    int num;
    int revNum;
    for(num = 1; num <= 100; num++){
        if(isPrime(num)){
            revNum = getReverse(num);
            if(revNum == num) continue;
            if(isPrime(revNum)) printf("%d\n", num);
        }
    }
    return 0;
}